gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/parameters/Key.java

```diff
@@ -39,6 +39,12 @@ public enum Key {
     PORTAL_TCP_PORT("portal.tcpPort", "4082", new HashSet<>(Arrays.asList(ENVIRONMENT, ORGANIZATION, SYSTEM))),
     PORTAL_KAFKA_DOMAIN("portal.kafkaDomain", "", new HashSet<>(Arrays.asList(ENVIRONMENT, ORGANIZATION, SYSTEM))),
     PORTAL_KAFKA_PORT("portal.kafkaPort", "9092", new HashSet<>(Arrays.asList(ENVIRONMENT, ORGANIZATION, SYSTEM))),
+    PORTAL_KAFKA_SASL_MECHANISMS(
+        "portal.kafka.saslMechanisms",
+        Arrays.asList("PLAIN", "SCRAM-SHA-256", "SCRAM-SHA-512").toString(),
+        List.class,
+        new HashSet<>(Arrays.asList(ENVIRONMENT, ORGANIZATION))
+    ),
```

Comment:
Have you tried with this syntax (like for PORTAL_HTTP_CORS_ALLOW_METHODS) ?
```suggestion
    PORTAL_KAFKA_SASL_MECHANISMS(
        "portal.kafka.saslMechanisms",
        "PLAIN;SCRAM-SHA-256;SCRAM-SHA-512",
        List.class,
        new HashSet<>(Arrays.asList(ENVIRONMENT, ORGANIZATION))
    ),
```